index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
+			const retryAfter = options !== undefined &&
+				['number', 'string'].includes(typeof options.retryAfter) ?
```

Comment:
There should be a way for users to disable this timeout. At the moment, this could be done by passing `Infinity`, but I think passing `false` might be more proper.

Reply:
I added another flag `retry` which can be set to true of false: ab00810a95cb9ac78855babcebd1f8c6febd4af0